client/sttp-client/src/main/scalajs/sttp/tapir/client/sttp/SttpClientInterpreterPlatformSpecific.scala

```diff
@@ -0,0 +1,5 @@
+package sttp.tapir.client.sttp
+
+trait SttpClientInterpreterPlatformSpecific {
```

Comment:
The convention in sttp is to use "extension" suffix, so I'd suggest to rename this to `SttpClientInterpreterExtensions`